docs/user-guide/wsl-support.md

```diff
@@ -51,6 +51,7 @@ Automatically detects and supports:
 - VS Code Insiders (`Code - Insiders`)
 - Windsurf (`Windsurf`)
 - Cursor (`Cursor`)
+- Kiro (`Kiro`)
```

Comment:
The VSCode Flavour detection is still relevant, but not sure if it belongs to this section.